package.json

```diff
@@ -152,12 +151,12 @@
     "chai-enzyme": "^0.5.1",
     "cheerio": "^0.20.0",
     "enzyme": "^2.9.0",
-    "eslint": "^3.19.0",
-    "eslint-config-airbnb": "^15.0.1",
-    "eslint-plugin-import": "^2.3.0",
+    "eslint": "^4.3.0",
+    "eslint-config-airbnb": "^15.1.0",
+    "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jest": "^20.0.3",
-    "eslint-plugin-jsx-a11y": "^5.0.3",
-    "eslint-plugin-react": "^7.0.1",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
```

Comment:
It is being used by `eslint-config-airbnb` and requires to be present in dependencies to run `eslint`.